industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/cdk2/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,8 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
```

Comment:
Could you please add the maestro version you used during the preparation (here and in the other preparation instruction files). Thanks!

Reply:
I have added this